src/cps/ast.rs

```diff
@@ -0,0 +1,130 @@
+//! This module defines and includes utilities for dealing with the intermediate CPS
+//! representation of Rust used by Liquid Rust.
+
+use rustc_span::Symbol;
+
+// TODO: Where should our Box<>es be?
+
+/// Each function in MIR is translated to a CpsFn
+#[derive(Debug)]
+pub struct CpsFn {
```

Comment:
What about `FnDef` for this one?
```suggestion
pub struct FnDef {
```